manifests/apache/vhost.pp

```diff
@@ -99,23 +99,27 @@
   Boolean $ldap_require_group               = $puppetboard::params::ldap_require_group,
   Optional[String] $ldap_require_group_dn   = undef,
   Hash $custom_apache_parameters            = {},
-) inherits ::puppetboard::params {
+  Hash $wsgi_daemon_process_options_extra   = {},
+) inherits puppetboard::params {
 
   $docroot = "${basedir}/puppetboard"
 
   $wsgi_script_aliases = {
     "${wsgi_alias}" => "${docroot}/wsgi.py",
   }
 
-  $wsgi_daemon_process_options = {
-    threads => $threads,
-    group   => $group,
-    user    => $user,
+  $wsgi_daemon_process_options_default = {
+    threads     => $threads,
+    group       => $group,
+    user        => $user,
+    python-home => '/srv/puppetboard/virtenv-puppetboard',
```

Comment:
There is a basedir parameter which is documented with " Base directory where to build puppetboard vcsrepo and python virtualenv." so should we use this parameter here?

Reply:
And the init.pp had a virtualenv_dir parameter

Reply:
thanks, fixed it